orquesta/machines.py

```diff
@@ -181,6 +181,25 @@
         events.TASK_CANCELED_WORKFLOW_ACTIVE: statuses.CANCELING,
         events.TASK_CANCELED_WORKFLOW_DORMANT: statuses.CANCELED
     },
+    statuses.RERUNNING: {
+        events.WORKFLOW_RUNNING: statuses.RUNNING,
+        events.WORKFLOW_FAILED: statuses.FAILED,
+        events.TASK_RUNNING: statuses.RUNNING,
+        events.TASK_SUCCEEDED_WORKFLOW_ACTIVE_INCOMPLETE: statuses.RUNNING,
+        events.TASK_SUCCEEDED_WORKFLOW_ACTIVE_COMPLETED: statuses.RUNNING,
+        events.TASK_SUCCEEDED_WORKFLOW_DORMANT_INCOMPLETE: statuses.RUNNING,
+        events.TASK_SUCCEEDED_WORKFLOW_DORMANT_COMPLETED: statuses.SUCCEEDED,
+        events.TASK_SUCCEEDED_WORKFLOW_DORMANT_PAUSED: statuses.PAUSED,
+        events.TASK_SUCCEEDED_WORKFLOW_DORMANT_CANCELED: statuses.CANCELED,
+        events.TASK_FAILED_WORKFLOW_ACTIVE: statuses.FAILED,
+        events.TASK_FAILED_WORKFLOW_DORMANT: statuses.FAILED,
+        events.TASK_REMEDIATED_WORKFLOW_ACTIVE_INCOMPLETE: statuses.RUNNING,
+        events.TASK_REMEDIATED_WORKFLOW_ACTIVE_COMPLETED: statuses.RUNNING,
+        events.TASK_REMEDIATED_WORKFLOW_DORMANT_INCOMPLETE: statuses.RUNNING,
+        events.TASK_REMEDIATED_WORKFLOW_DORMANT_COMPLETED: statuses.SUCCEEDED,
+        events.TASK_REMEDIATED_WORKFLOW_DORMANT_PAUSED: statuses.PAUSED,
+        events.TASK_REMEDIATED_WORKFLOW_DORMANT_CANCELED: statuses.CANCELED,
+    },
```

Comment:
Remove if these events are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.